PowerMode/PowerMode.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.VSSDK.BuildTools.15.0.26201\build\Microsoft.VSSDK.BuildTools.props" Condition="Exists('..\packages\Microsoft.VSSDK.BuildTools.15.0.26201\build\Microsoft.VSSDK.BuildTools.props')" />
+  <Import Project="..\packages\Microsoft.VSSDK.BuildTools.17.0.5232\build\Microsoft.VSSDK.BuildTools.props" Condition="Exists('..\packages\Microsoft.VSSDK.BuildTools.17.0.5232\build\Microsoft.VSSDK.BuildTools.props')" />
```

Comment:
I found this updated the VSSDK and target framework version. So I think this might break the usage for the early version of Visual Studio.

Can you please help me to update it or test it out on the earlier version of VS?

Reply:
You're right. I'll test that out with VS2015

Reply:
Alright, I reverted some changes I made so VS2017 can build the projects. The downside is, that they won't build with VS2022 since it requires the "Product Architecture" property for all installation targets inside the vsixmanifest.
Such as:
```xml
<InstallationTarget Version="[15.0,18.0)" Id="Microsoft.VisualStudio.Community">
            <ProductArchitecture>amd64</ProductArchitecture>
</InstallationTarget>
```

In my opinion, this doesn't matter too much since I think most people get your extension from the VS marketplace anyway. If they want to build the code independently, they just have to set the architecture settings.